clients/go/cmd/zbctl/internal/commands/cancel.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
No Idea why go fmt touched this or was it my editor 🤔 .

Reply:
Are these tabs? Do you have maybe different setting for tabs 🤔 like converting to 4 spaces or something.